lint.swift

```diff
@@ -0,0 +1,547 @@
+        matchingExamples: [
+            """
+            guard
+                let collection = viewModel.myCollection(),
+                !collection.compactMap({ OtherType($0) }).isEmpty() else {
+                return
+            }
+            """,
+            """
+            guard let collection = viewModel.myCollection(),
+                !collection.compactMap({ OtherType($0) }).isEmpty() else {
+                return
+            }
+            """
+        ],
+        nonMatchingExamples: [
+            """
+            guard
+                let collection = viewModel.myCollection(),
+                !collection.compactMap({ OtherType($0) }).isEmpty()
+            else {
+                return
+            }
+            """,
+            """
+            guard let collection = viewModel.myCollection(),
+                !collection.compactMap({ OtherType($0) }).isEmpty()
+            else {
+                return
+            }
+            """
+        ],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles]
+    )
+
+    // MARK: MultilineGuardStart
+    try Lint.checkFileContents(
+        checkInfo: "MultilineGuardStart: Always start a multiline guard via `guard` then a line break and all expressions indented.",
+        regex: #"guard([^\n]*,\s*\n)+[^\n]*\s*else\s*\{"#,
+        matchingExamples: [
+            """
+            guard let collection = viewModel.myCollection(),
+                !collection.compactMap({ OtherType($0) }).isEmpty()
+            else {
+                return
+            }
+            """,
+            """
+            guard let collection = viewModel.myCollection(),
+                !collection.compactMap({ OtherType($0) }).isEmpty() else {
+                return
+            }
+            """
+        ],
+        nonMatchingExamples: [
+            """
+            guard
+                let collection = viewModel.myCollection(),
+                !collection.compactMap({ OtherType($0) }).isEmpty()
+            else {
+                return
+            }
+            """,
+            """
+            guard
+                let collection = viewModel.myCollection(),
+                !collection.compactMap({ OtherType($0) }).isEmpty() else {
+                return
+            }
+            """
+        ],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles]
+    )
+
+    // MARK: NavigationControllerVariableNaming
+    try Lint.checkFileContents(
+        checkInfo: "NavigationControllerVariableNaming: Always name your navigation controller variables with the suffix `NavCtrl` or just `navCtrl`.",
+        regex: #"(var|let) +(nc|navigationcontroller|navc|ncontroller|navcontroller)[ :][^\n=]*=\i"#,
+        matchingExamples: ["let nc =", "var navigationController =", "let navc =", "let ncontroller =", "var nc: MyNavigationController =", "let navController: MyNavigationController<T> = "],
```

Comment:
Codacy found an issue: [Always name your view controller variables with the suffix `ViewCtrl`.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)